packages/@ngtools/webpack/src/angular_compiler_plugin.ts

```diff
@@ -192,6 +192,12 @@ export class AngularCompilerPlugin implements Tapable {
       tsConfigJson.exclude = tsConfigJson.exclude.concat(options.exclude);
     }
 
+    // Add extra includes.
+    if (options.hasOwnProperty('include')) {
```

Comment:
What about `files`?

Reply:
We already had `includes` as part of the options so it makes more sense than `files` imho.